scala-package/pom.xml

```diff
@@ -30,12 +30,13 @@
     <connection>scm:git:git@github.com:apache/incubator-mxnet.git</connection>
     <developerConnection>scm:git:git@github.com:apache/incubator-mxnet.git</developerConnection>
     <url>https://github.com/apache/incubator-mxnet</url>
-    <tag>mxnet-parent_2.11-1.2.0</tag>
+    <tag>HEAD</tag>
   </scm>
 
   <properties>
     <scala.version>2.11.8</scala.version>
     <scala.binary.version>2.11</scala.binary.version>
+    <build.platform />
```

Comment:
Why we have it here, it seemed empty?

Reply:
this is passed from the Makefile. ie., to say no default value